go/libkb/login_state.go

```diff
@@ -200,7 +211,10 @@ func (s *LoginState) GetPassphraseStream(ui SecretUI) (pps *PassphraseStream, er
 	if pps != nil {
 		return pps, nil
 	}
-	if err = s.verifyPassphraseWithServer(ui); err != nil {
```

Comment:
the new code on line 214 does what verifyPassphraseWithServer does, but passes in a username (instead of relying on G.Env.GetUsername().